src/components/Progress.tsx

```diff
@@ -30,7 +30,7 @@ export default (props: ProgressProps) => {
 
 const styles: any = {
     inner: {
-        background: '#fff',
+        backgroundImage: 'linear-gradient(to right, #5544e7 , #69b9fb)',
```

Comment:
Hey @tonycc0606 , this would affect all existing users of the component. If you want this feature to be added, please make it configurable via a prop e.g. `progressBarStyles`.